cluster-operator/src/test/resources/kafka-2.0.0-config-model.json

```diff
@@ -0,0 +1,808 @@
+{
```

Comment:
I guess this file should be deleted.

Reply:
A lot of the tests are hard coded to use 2.0 (and also 2.1 in some cases). It would be quite a lot of work to change them to be version-agnostic. While I think we should do that it's simpler for this PR to just add a 2.0 config model.